flow/project.py

```diff
@@ -3796,14 +4150,22 @@ def _select_jobs_from_args(self, args):
                 "Cannot provide both -j/--job-id and -f/--filter or --doc-filter in combination.")
 
         if args.job_id:
-            try:
-                return [self.open_job(id=job_id) for job_id in args.job_id]
-            except KeyError as error:
-                raise LookupError("Did not find job with id {}.".format(error))
-        else:
+            aggregate = set()  # set in order to avoid duplicate ids
+            for id in args.job_id:
+                if id.startswith('agg'):
+                    aggregate.add(self._generate_aggregate_from_id(id))
+                    continue
+                try:
+                    aggregate.add((self.open_job(id=id),))
+                except KeyError as error:
+                    raise LookupError("Did not find job with id {}.".format(error))
+            return list(aggregate)
+        elif args.filter or args.doc_filter:
             filter_ = parse_filter_arg(args.filter)
             doc_filter = parse_filter_arg(args.doc_filter)
-            return JobsCursor(self, filter_, doc_filter)
+            return list({str(job) for job in JobsCursor(self, filter_, doc_filter)})
```

Comment:
Why is this a list of ids?

Reply:
I'm sorry, I missed this part of the implementation. 
I will change it